### ISSUE TITLE
[Tech Debt] 526 Claims: Generic Claim Creation Failure During Establish Claim

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#11352]
## Issue Description
the response we see from EVSS:
`form526.submit.establishClaim.serviceError: Claim creation failed. System error.`

The message that displays in our rake report:
`External service unavailable`

[example sentry error ](http://sentry.vfs.va.gov/organizations/vsp/issues/49331/events/e6a272f16523404c8baeca04ea0d348e/?project=3&statsPeriod=14d)

This is an error that EVSS gets from VBMS.  EVSS gets extra detail from VBMS that does not get passed on to us that is highly relevant in troubleshooting. 

here is an example of the level of detail that VBMS returns to EVSS:
https://dsva.slack.com/archives/CK466V65V/p1617215319009900?thread_ts=1617115755.000400&cid=CK466V65V

TL;DR EVSS is getting an error from VBMS - we do not get the details but EVSS does

**Comment by annaswims** [comment]
we don't see this error any more

**Comment by annaswims** [comment]
actually, we still see this error

**Comment by annaswims** [comment]
This error affected 220 people in the last month.

**Comment by cnemeth** [comment]
1. The `PIFInUse` issue will be fixed by _EVSS_ team. However, there is no timeframe for when the fix is going to be implemented.
2. In the meantime, `vets-api` should implement a (temporary) solution, such that,
  - stop resubmitting the claim if the _JSON response_ contains the `PIFInUse` key, and, 
  - stop displaying the misleading message to users stating that their " ... claims is taking longer than expected".

**Comment by cnemeth** [comment]
Closing ticket. As per team discussion this morning, there is nothing to do on the backend.
